cardscontainer.cpp

```diff
@@ -64,3 +64,8 @@ void CardsContainer::showCard(QString path, QPushButton* button){
     button->setIcon(buttonIcon);
     button->setIconSize(button->rect().size());
 }
+CardsContainer::~CardsContainer(){
+    for(auto& it : cards_){
```

Comment:
a nie można trzymać unique_ptr w cards_, żeby nie było się trzeba martwić zwalnianiem pamieci ?|

Reply:
potrzebujemy raw_ptr do łączenia przycisków ze slotami. Gdy robimy to za pomocą .get() wywala dziwne błędy